test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -198,17 +205,17 @@ Describe "Test-Connection" -tags "CI" {
     # TODO: We skip the MTUSizeDetect tests on Unix because we expect 'TtlExpired' but get 'TimeOut' internally from .Net Core
     # Skipping on VSTS in Windows due to `TimedOut`
     Context "MTUSizeDetect" {
-        It "MTUSizeDetect works" -Pending:(!$isWindows -or (Test-IsVstsWindows)) {
-            $result = Test-Connection $realName -MTUSizeDetect
+        It "MTUSizeDetect works" -Pending:(!$isWindows) {
```

Comment:
Can you double check if this still needs to be maked as Pending. It worked on my Ubuntu 18.04 VM.

Reply:
it's definitely failing on my mac, I'll mark it pending for `$IsMacOS`